src/components/apibutton.js

```diff
@@ -81,21 +85,15 @@ export default function ApiButton(props) {
     }
 
     const transformedStateToQuery = (arr) => {
-        return "?" + arr.map(x => {
```

Comment:
Removing the question mark from here, and bringing it to stateFormatter very questionable. You've increased the amount of places you need to write the ? while also removing it from a place it's guaranteed to be added to every query string.